lib/src/hooks/useDynamicForm.tsx

```diff
@@ -145,6 +158,7 @@ export function useDynamicForm(
     });
     const [state, setState] = useState<DynamicFormInternalState>(initialState);
     const [errorMessages, setErrorMessages] = useState<DynamicFormErrors>({});
+    const [touched, setTouched] = useState<boolean>(false);
```

Comment:
maybe can refactor to the more commonly use term `dirty`?

Reply:
Ah yes dirty is more accurate! thanks!